tests/v2/waku_rln_relay/test_wakunode_rln_relay.nim

```diff
@@ -51,7 +51,7 @@ procSuite "WakuNode - RLN relay":
     await node1.mountRlnRelay(WakuRlnConfig(rlnRelayDynamic: false,
       rlnRelayPubsubTopic: rlnRelayPubSubTopic,
       rlnRelayContentTopic: contentTopic,
-      rlnRelayMembershipIndex: some(MembershipIndex(1)),
+      rlnRelayCredIndex: MembershipIndex(1),
```

Comment:
Would it make sense to also rename the type `MembershipIndex`?

Reply:
This looks like it fell through the sieve, it should actually be <index>.uint instead. fixed in https://github.com/waku-org/nwaku/pull/1777/commits/956a63c8e20bd326626b1e16e2d938c696e28c1d